src/lib/frontend/typechecker.ml

```diff
@@ -2148,17 +2148,21 @@ let rec push_pop_env f n s =
   if n <= 1 then
     f s
   else
-    let _ = f s in
+    let _env = f s in
```

Comment:
It's not env in case `f` is `Stack.push` but of type `unit`.

Reply:
That's why `let _ = ` was a good solution for me :)

Reply:
Ok, let's keep `let _ = ...`